recipe/meta.yaml

```diff
@@ -70,6 +71,7 @@ requirements:
     - openssh  # [mpi == 'openmpi']
     - mkl 2020.0  # [win]
     - intel-openmp 2020.0  # [win]
+    - libssh2 1.8.*
```

Comment:
I'm not sure forcing this here is the better way to fix this. Pretty much everything that depends indirectly on `libssh2` (or `curl`) will be broken due to https://github.com/conda-forge/conda-forge-repodata-patches-feedstock/pull/46, right?

Reply:
I guess this will also be reversed if the repodata are run again.

Reply:
Yep.

Reply:
Yes, this was just a test, but the bot merged it

Reply:
This is causing trouble and I guess I can take it out now?